resources/META-INF/plugin.xml

```diff
@@ -238,6 +238,7 @@
         <internalFileTemplate name="Magento PHP Form Generic Button Block Class"/>
         <internalFileTemplate name="Magento Entity New Action Controller Class"/>
         <internalFileTemplate name="Magento New Entity Layout XML"/>
+        <internalFileTemplate name="Magento Delete Entity By Id Command Model"/>
```

Comment:
I wonder why we need the word `Model` here. It's just a command, isn’t it?

Reply:
@eduard13 @coderimus what do you think guys?

Reply:
💯 agree, `Model` word is extra

Reply:
@VitaliyBoyko agree that the `Model` is an extra word.

Reply:
thanks 🙏